presto-oracle/src/test/java/io/prestosql/plugin/oracle/TestingOracleServer.java

```diff
@@ -87,7 +87,7 @@ public TestingOracleServer()
     @Override
     public String getJdbcUrl()
     {
-        return "jdbc:oracle:thin:@" + getHost() + ":" + getOraclePort() + ":" + getSid();
+        return "jdbc:oracle:thin:@" + getHost() + ":" + getOraclePort() + ":" + getSid() + "?remarksReporting=true";
```

Comment:
So with this approach comments would be supported only if user of the connector adds `remarksReporting=true` into db URL.
I would suggest that we add the parameter in io.prestosql.plugin.oracle.OracleClientModule#connectionFactory().

Reply:
Oh - I have just read your comment on the PR :) Let's leave it as is. Can you add a note to the documentation?

Reply:
is this needed? 
oracle always have a comment, empty ("") or with some text..

Reply:
As far as I tested on the current `TestingOracleServer`, it is needed.

Reply:
@losipiuk Added a note to the documentation.